app/jobs/cron/expired_prefilled_dossiers_deletion_job.rb

```diff
@@ -0,0 +1,7 @@
+class Cron::ExpiredPrefilledDossiersDeletionJob < Cron::CronJob
+  self.schedule_expression = "every month at 3:00"
+
+  def perform
+    Dossier.prefilled.state_brouillon.where("updated_at < ?", 1.month.ago).destroy_all
```

Comment:
```suggestion
    Dossier.prefilled.state_brouillon.where(user_id: nil, updated_at: ..1.month.ago).destroy_all
```

Il faut détruire que les dossiers pas attachés aux usagers. J'ai l'impression aussi qu’un mois c'est assez long. À moins que tu aies un cas d'usage déjà en tête, je commencerai avec quelque chose de plus court (quelques jours ?) et on pourra augmenter si l'usage montre que c'est nécessaire.

`destroy_all` c'est très bien ici

Reply:
Deal ! Va pour 1 semaine :)

J'attends que le POST soit mergé et je repasse dessus rapidement. Merci !

Reply:
C'est mergé :)